pagopa/io-app
# chore(Cross): [IOAPPX-439] Enable the iridescent trust mark for all

## Short description
This PR enables the iridescent trust mark for all. It was previosuly tested internally through the experimental DS feature flag.
To learn more about the effect:
* https://github.com/pagopa/io-app/pull/6179

## List of changes proposed in this pull request
- Update `react-native-skia` to the latest version to take advantage of Android performance improvements

## How to test
Go to the credential detail page with experimental DS disabled and check if everything is OK
